ts/src/transports/fetch.ts

```diff
@@ -22,14 +22,15 @@ class Fetch implements Transport {
     this.options = transportOptions;
   }
 
-  pump(readerArg: ReadableStreamReader, res: Response): Promise<void | Response> {
+  pump(readerArg: ReadableStreamReader, res: Response) {
     this.reader = readerArg;
```

Comment:
Could you explicitaly type the return value, please?

Reply:
Pump no longer returns anything. The method now matches the [send](https://github.com/improbable-eng/grpc-web/blob/master/ts/src/transports/fetch.ts#L47) method within the class.

The previously returned promise was never actually used anywhere. The pump method was only called recursively, or from within the `send` method; and send has no return value.

Reply:
> All methods within this class that aren't returning anything are missing the void return type

🤔 Apologies I assumed we had explicit typing. 

Given that we do not this boils down to personal preference so feel free to disregard my comment.